examples/features/loops.effekt.md

```diff
@@ -0,0 +1,46 @@
+# Loops
+
+In Effekt, there is only a `while` loop expression built-in (that is completely standard).
```

Comment:
Nitpick: the formulation of this sentence is pretty weird and it took me a while to understand it.
```suggestion
Effekt includes a (completely standard) built-in `while` loop expression as its sole native looping construct.
```